src/test/java/org/influxdb/InfluxDBTest.java

```diff
@@ -1457,6 +1457,13 @@ public String call() throws Exception {
     Assertions.assertTrue(MyInfluxDBBean.OKHTTP_BUILDER.interceptors().isEmpty());
   }
 
+  @Test
+  public void testQueryPostWithGZIPCompression() {
+    this.influxDB.enableGzip();
+    String command = String.format("CREATE DATABASE db_gzip_%d", System.currentTimeMillis());
+    this.influxDB.query(new Query(command, null, true));
```

Comment:
shouldn't we test the result meets some expectations ?

Reply:
I've added a test to check the result of `CREATE DATABASE ...`.